lib/number.js

```diff
@@ -116,6 +116,7 @@ export function getNumberDescriptor() {
 	let percentPattern = '{number} %';
 	let percentNegativePattern = '-{number} %';
 	switch (baseLanguage) {
+		case 'es':
```

Comment:
Same here -- we were treating Mexico differently, but inspecting the data the monolith has for `es-MX` I noticed it matches `es-ES` so combining them.